libs/component-sample/src/lib/feature/subheader-sample/subheader-sample.component.html

```diff
@@ -1,23 +1,45 @@
-<sds-subheader>
-  <!-- =============== TITLE =============== -->
+<div class="container">
+  <sds-subheader>
+    <!-- =============== TITLE =============== -->
 
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque tempor a enim
-  ac molestie. Nam ut risus magna. Quisque at pulvinar sem, id dictum ipsum. Nam
-  commodo rhoncus mauris, id gravida lorem congue nec. Sed dictum, tellus sed
-  commodo sempertr
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque tempor a enim
+    ac molestie. Nam ut risus magna. Quisque at pulvinar sem, id dictum ipsum. Nam
+    commodo rhoncus mauris, id gravida lorem congue nec. Sed dictum, tellus sed
+    commodo sempertr
 
-  <!-- =============== PRIMARY/SECONDARY BUTTONS =============== -->
-  <sds-subheader-buttons
-    [model]="subheader.buttons"
-    (clicks)="log($event)"
-  ></sds-subheader-buttons>
+    <!-- =============== PRIMARY/SECONDARY BUTTONS =============== -->
+    <sds-subheader-buttons [model]="subheader.buttons" (clicks)="log($event)"></sds-subheader-buttons>
 
-  <!-- =============== SEARCH =============== -->
-  <sds-subheader-search (searchTerm)="log($event)"> </sds-subheader-search>
+    <!-- =============== SEARCH =============== -->
+    <sds-subheader-search (searchTerm)="log($event)"> </sds-subheader-search>
 
-  <!-- =============== ACTIONS BUTTONS =============== -->
-  <sds-subheader-actions
-    [model]="subheader.actions"
-    (clicks)="log($event)"
-  ></sds-subheader-actions>
-</sds-subheader>
+    <!-- =============== ACTIONS BUTTONS =============== -->
+    <sds-subheader-actions [model]="subheader.actions" (clicks)="log($event)"></sds-subheader-actions>
+    <!-- =============== DRAWER CONTENT =============== -->
+    <sds-subheader-drawer [drawerContentTemplate]="temp"></sds-subheader-drawer>
```

Comment:
Can we rename temp to template (throughout the PR)? We don't want to confuse temporary with template.

Reply:
changed to contentTemplate